# (no request fields or constants)

---

# response constants
int8 DIRECT_DRIVE=0
int8 AUTONOMY=1
int8 IDLE=2

# response fields
int8 state
